scalafix/rules/src/main/scala/rsc/rules/RscCompat.scala

```diff
@@ -11,39 +11,39 @@ import rsc.rules.syntax._
 import scala.meta._
```

Comment:
Also, please submit future pull requests against `topic/better` in my repository. This will avoid extraneous commits in the "Commits" tab.

Reply:
I wondered why that happened, thanks for pointing out I PR'd against master accidentally.